client/src/components/Modals/EditInterests/styles.scss

```diff
@@ -0,0 +1,80 @@
+@import '../../../styles/colors.scss';
```

Comment:
I notice that this SCSS file is identical to the one in the teams modal. Now that the teams stuff is merged in, can you find a way to make this less copy paste?